youtube_dl/extractor/filemoon.py

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    decode_packed_codes,
+    js_to_json,
+)
+
+
+class FileMoonIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?filemoon\.sx/./(?P<id>\w+)(?:/(?P<title>.+))*'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?filemoon\.sx/./(?P<id>\w+)'
```

Reply:
Not sure about this change, I know there are links with this form (filemoon/id/title) but I cannot find a very good example

Reply:
If there are links like that, the final part of the URL path will be picked up by `_generic_title()`, so replicating what you had, or I hope so.